plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -159,6 +159,8 @@
 {
     private static final Logger log = Logger.get(IcebergMetadata.class);
 
+    public static final String TRINO_QUERY_ID_NAME = "trino_query_id";
```

Comment:
I moved the constant into this class so that it would be in the `trino-iceberg` project only.

However, I'm not sure this is really the best place for it.